packages/react/src/drawer/docs/overview.mdx

```diff
@@ -72,6 +72,46 @@ The Drawer can be closed by either:
 
 When closing the Drawer, focus should be returned to the button that opened the Drawer.
 
+## Drawer with muted overlay
```

Comment:
I would move this section under the widths section since we want to promote this less

Reply:
Moved